scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,180 @@
+                elif 'Incremental' in statvar:
+                    sv_full_name = ('incrementalCount_MedicalConditionIncident_'
+                                    'COVID_19_' + statvar)
+                else:
+                    sv_full_name = ('count_MedicalConditionIncident_COVID_19_'
+                                    + statvar)
+                f_out.write(TEMPLATE.format_map({'index': idx,
+                    'SVname': statvar, 'SVFullName': sv_full_name,
+                    'geoNode': geo_node}))
+                idx += 1
+
+    def _translate(self):
+        """translate coloumn names from italian to english"""
+        it2en = {'data': 'Date', 'stato': 'State',
+                 'codice_regione':'RegionCode',
+                 'denominazione_regione': 'RegionName',
+                 'codice_provincia': 'ProvinceCode',
+                 'denominazione_provincia': 'ProvinceName',
+                 'sigla_provincia': 'ProvinceAbbreviation',
+                 'lat': 'Latitude', 'long': 'Longitude',
+                 'ricoverati_con_sintomi': 'HospitalizedsWithSymptoms',
+                 'terapia_intensiva': 'IntensiveCare',
+                 'totale_ospedalizzati': 'Hospitalized',
+                 'isolamento_domiciliare': 'PeopleInHomeIsolation',
+                 'totale_positivi': 'ActiveCase',
+                 'variazione_totale_positivi': 'IncrementalPositiveCase',
+                 'nuovi_positivi': 'IncrementalActiveCase',
+                 'dimessi_guariti': 'CumulativeRecovered',
+                 'deceduti': 'CumulativeDeath',
+                 'totale_casi':'CumulativePositiveCase',
+                 'tamponi': 'CumulativeTestsPerformed',
+                 'casi_testati': 'CumulativeTestedPeople',
+                 'casi_da_sospetto_diagnostico': 'PositiveCasesFromClinicActivity',
+                 'casi_da_screening': 'PositiveCasesFromSurveyAndTest'}
+        self.data = self.data.rename(columns=it2en)
+
+    def set_location(self):
+        pass
+
+    def geo_template(self):
+        pass
+
+class PcmDpcNational(PcmDpc):
+    """subclass processing national data"""
+    def __init__(self):
+        self.csvpath = ('https://raw.githubusercontent.com/pcm-dpc/COVID-19/'
+                        'master/dati-andamento-nazionale/dpc-covid19-ita-and'
+                        'amento-nazionale.csv')
+        self.name = "dpc-covid19-ita-national-trend"
+        self.stat_vars = ['HospitalizedsWithSymptoms', 'IntensiveCare',
+                          'Hospitalized', 'PeopleInHomeIsolation', 'ActiveCase',
+                          'IncrementalPositiveCase', 'IncrementalActiveCase',
+                          'CumulativeRecovered', 'CumulativeDeath',
+                          'CumulativePositiveCase', 'CumulativeTestsPerformed',
+                          'CumulativeTestedPeople']
+
+    def set_location(self):
+        assert (self.data['State'] == 'ITA').all()
```

Comment:
Ok for now, but I was thinking--since the columns for nat'l and regional are the same, if you kept the same format, they could use the same template MCF. Is that correct?

Reply:
Right. Or we can combine the national and regions level into a single file.

Reply:
Actually, one concern is that the field for "observationAbout" maybe different, or the field for "typeOf" of the geo node would be different. Maybe "typeOf" is not necessary if we're specifying the dcids?